distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1208,6 +1205,11 @@ type Column
          as "3 Dec 2011". See https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/time/format/DateTimeFormatter.html
          for a complete format specification.
 
+         Note that the format string can specify decimal point and digit
+         separators, but these characters are interpreted in the context of the
+         Locale used. The format string specifies their location, but the Locale
+         has the final decision about which characters are used.
```

Comment:
Can we get an example for this? This description is _perfect_ but an example can speak a thousand words.

Just one with FR locale copied from tests would be perfect IMO.

Reply:
Added an example.